src/cic/repl.clj

```diff
@@ -81,7 +81,10 @@
 
 (defn prepare-periods
   [{:keys [periods] :as model-inputs} episodes-extract-date rewind-years]
-  (let [project-from (time/years-before episodes-extract-date rewind-years)
+  (let [episodes-extract-date (if (string? episodes-extract-date)
```

Comment:
slightly worried about the type being unknown there and thus needing a  check, but hopefully it does the guarding we need.

Reply:
I've been meaning to ask why the `string?` check. The reader macro `#date` is supposed to ensure we don't need to do this kind of parsing ourselves https://github.com/MastodonC/witan.cic/blob/master/data/demo/config.edn#L18

Reply:
Yeah my apologies, I was previously running stuff from the old repl.clj namespace, at which point I was having trouble reading in a config file with a reader macro `#date`, hence why I insrted this to deal with that. It can be removed

Reply:
This branch is actually redundant now so I'll delete it